contracts/lib/OrderFulfiller.sol

```diff
@@ -193,7 +193,7 @@ contract OrderFulfiller is
          */
 
         // Declare a nested scope to minimize stack depth.
-        {
+        unchecked {
```

Comment:
Somewhat arguable if the scope should be so wide. In the previous code, it was rather clear what was `unchecked`.